src/ui/components/VariantSelector.tsx

```diff
@@ -39,28 +39,38 @@ export function VariantSelector(props: { variants: { id: string; name: string }[
 		}
 	}, [selectVariant, variants]);
 
+	const generatedId = useId();
+
 	return (
-		<div className="my-4">
-			<div className="sr-only">Variants</div>
+		<div className="my-4" role="radiogroup" aria-labelledby={`${generatedId}-title`}>
```

Comment:
Maybe this should be a `fieldset` with `legend` instead? So we can avoid generating the id.

Reply:
Nice suggestion. I also considered to use that tag, but I wasn't sure if can I use it outside of the form.